visualizer/actionButton.js

```diff
@@ -279,4 +366,17 @@
     document.querySelector('#operations').innerHTML = '';
     document.querySelector('#attributes').innerHTML = '';
   });
+
+  // Hide the action context menu when Esc or Enter is pressed, any click happens, or another
+  // context menu is brought up.
+  document.addEventListener('click', hideContextMenu);
```

Comment:
I duplicated the code here to hide the context menu for operation button, maybe we should put the hiding code together in a separate file?